assets.conf

```
@@ -65,3 +65,9 @@ exec = python3 ./tools/soko/soko_tmx_preprocessor.py "%i" "%o"
 [.txt]
 outExt = txt
 func = text
+
+; CH32V003 files
+[.cfun]
+outExt = cfun
```

This was the regeneration problem. The input is a `.cfun` file and the output is a `.cfun.bin`. It was looking for `.cfun` in the output folder, not finding it, and copying every build.
```suggestion
outExt = cfun.bin
```